Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed Inspector Window being refreshed all the time when a PlayerInput component is present with Invoke Unity Events nofication mode chosen [ISXB-1448](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1448)
 - Fixed an issue where an action with a name containing a slash "/" could not be found via `InputActionAsset.FindAction(string,bool)`. [ISXB-1306](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1306).
 - Fixed Gamepad stick up/down inputs that were not recognized in WebGL. [ISXB-1090](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1090)
+- Fixed Action with custom processor serialises enums by index rather than by value. [ISXB-1474](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1474)
```

Comment:
This URL is public, great. Would generally recommend explaining what fix does and previous symptom, e.g. "Fixed an issue that caused input processors with enum properties to incorrectly serialise by index instead of by value."